gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx

```diff
@@ -22,15 +22,15 @@ If you're already using [Element](https://element.io), you can browse all of the
 ✓ Public Chats  
 ✗ Private Chats  
 ✗ 1:1 Chats  
-✓ Native feeling usernames/avatar on both sides (virtual users)
-✓ Replies/Threaded conversations
-✓ Formatted Messages
-✓ Mentions
-✓ Edits
-✓ Redactions/Deletions
-✓ Status `/me` messages
+✓ Native feeling usernames/avatar on both sides (virtual users)  
+✓ Replies/Threaded conversations  
+✓ Formatted Messages  
+✓ Mentions  
+✓ Edits  
+✓ Redactions/Deletions  
+✓ Status `/me` messages  
 ✓ Media/Files  
-✗ Reactions
+✗ Reactions  
```

Comment:
I guess the key is double spaces after everything 🤷

Reply:
it is, markdown, sorry I didn't check on this!

Reply:
Ahh didn't know this trick. Thanks for the markdown hint!

> When you do want to insert a <br /> break tag using Markdown, you
end a line with two or more spaces, then type return.
>
> https://daringfireball.net/projects/markdown/syntax#p